charts/gardener/operator/templates/role.yaml

```diff
@@ -0,0 +1,140 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: gardener.cloud:system:gardener-operator
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app: gardener
+    role: operator
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - events
+  verbs:
+  - create
+  - patch
+  - update
```

Comment:
I believe we need permissions for `events` in `default`, as events for cluster-scoped resources (e.g. `Garden`) are recorded in the `default` namespace: [ref](https://github.com/kubernetes/kubernetes/blob/57eb5d631ccd615cd161b6da36afc759af004b93/staging/src/k8s.io/client-go/tools/record/event.go#L375-L379)